app/src/main/assets/vl1.svg

```diff
@@ -922,5 +922,7 @@ DUE TO MISSING FLOOR PLANS WITH ROOM NUMBERS, THE ROOMS HAVE BEEN ID FROM VL 101
           d="M23 10V13.9844H11V10H23ZM23.2812 18.7188C23.4688 18.9062 23.7031 19 23.9844 19C24.2656 19 24.5 18.9062 24.6875 18.7188C24.9062 18.5312 25.0156 18.2969 25.0156 18.0156C25.0156 17.7344 24.9062 17.5 24.6875 17.3125C24.5 17.0938 24.2656 16.9844 23.9844 16.9844C23.7031 16.9844 23.4688 17.0938 23.2812 17.3125C23.0938 17.5 23 17.7344 23 18.0156C23 18.2969 23.0938 18.5312 23.2812 18.7188ZM20.9844 25.9844V21.0156H13.0156V25.9844H20.9844ZM23.9844 15.0156C24.7969 15.0156 25.5 15.3125 26.0938 15.9062C26.6875 16.5 26.9844 17.2031 26.9844 18.0156V24.0156H23V28H11V24.0156H7.01562V18.0156C7.01562 17.2031 7.3125 16.5 7.90625 15.9062C8.5 15.3125 9.20312 15.0156 10.0156 15.0156H23.9844Z"
           fill="#DA3636" stroke-width="0" />
     </svg>
+    <svg x="175" y="950" id="entrance">
```

Comment:
This isn't the correct entrance point.
I will update the SVG and commit it to the proper destination.